modules/ROOT/pages/indexes-for-search-performance.adoc

```diff
@@ -463,11 +526,20 @@ Only one relationship type lookup index can exist at a time.
 ===== Create a token lookup index only if it does not already exist
 
 If it is not known whether an index exists or not, add `IF NOT EXISTS` to ensure it does.
+Giving the name directly or through a parameter makes no difference.
```

Comment:
I don't think we need to write this. This is sensible enough that it should be expected. Mentioning this just causes the reader to question whether there are cases where parameter vs literal would make a difference.

Reply:
I think I added it here just cause I couldn't add a new lookup index example, so therefore I also added the comment but I can remove the comment